### ISSUE TITLE
`Referrer-Policy` changes within the `.htaccess` file will not be applied.

### ISSUE THREAD

**Issue by iPaat** [nextcloud/server#14374]
`Referrer-Policy` changes within the `.htaccess` file will not be applied.

### Steps to reproduce
1. Change referrer policy in your `.htaccess` file from `no-referrer` to `strict-origin-when-cross-origin`.
```
Header set Referrer-Policy "strict-origin-when-cross-origin"
```
2. Place somewhere a `.php` file on a webserver with the following contents:
```php
<?php

var_dump($_SERVER);
```
3. Use the browser development tools to place a link on `DOMAIN.TLD/login` which points to your `.php` file on your webserver.
4. Click on that link and search for the `HTTP_REFERER` key in the output that the `.php` file you created produces.

### Expected behaviour
You should be able to see the `HTTP_REFERER` field.

### Actual behaviour
There is no `HTTP_REFERER` field.

### Server configuration

**Operating system**:
Linux

**Web server**:
Apache

**Nextcloud version:** (see Nextcloud admin page)
15.0.4

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**Are you using encryption:** yes

### Client configuration
**Browser:**
Firefox 66.0b10 & 65.0.1  
Chrome 72.0.3626.119

**Operating system:**
Windows

**Comment by kesselb** [comment]
How is this related to nextcloud?

**Comment by DominikWA** [comment]
I had the same problem, after a quick digging in I found out that there are meta tags that also sets the referral-policy. Here you can find my fix https://github.com/nextcloud/server/pull/14318.

Best,
Dominik

**Comment by kesselb** [comment]
> I had the same problem, after a quick digging in I found out that there are meta tags that also sets the referral-policy. Here you can find my fix #14318.

This could be the reason :+1:

**Comment by iPaat** [comment]
This helped, thanks a lot.

Hopefully it will be merged soon so that my source code doesn't deviate from the original.

Are there any security risks if I remove the meta tags as suggested?
